docs/source/researcher_resources.rst

```diff
@@ -0,0 +1,116 @@
+for population growth. It also contains a useful tutorial on interacting with data artifacts.
+
+- `Repository <https://github.com/ihmeuw/vivarium_public_health>`_
+- `Documentation <https://vivarium.readthedocs.io/projects/vivarium-public-health/en/latest/>`_
+
+
+Vivarium Inputs
+---------------
+
+Vivarium Inputs contains the tools needed to create data artifacts from GBD data. The data artifact is a useful notion,
+you can think of it as all of the data needed to run a simulation, packaged into one file. This package contains useful
+functions for pulling raw GBD data as well as generating data as it is used in a simulation, which are vitally important
+for vetting.
+
+- `Repository <https://github.com/ihmeuw/vivarium_inputs>`_
+- `Documentation <https://vivarium.readthedocs.io/projects/vivarium-inputs/en/latest/>`_
+
+
+Vivarium Cluster Tools
+----------------------
+
+Vivarium Cluster Tools contains command line tools for running Vivarium simulations in parallel using IHME's cluster.
+It contains documentation describing how this is done, as well as how YAML files and branches files work.
+
+- `Repository <https://github.com/ihmeuw/vivarium_cluster_tools>`_
+- `Documentation <https://vivarium-cluster-tools.readthedocs.io/en/latest/)>`_
+
+
+GBD Mapping
+-----------
+
+GBD Mapping is a python package that contains all the things modeled by the GBD and their relationships. This means
+things like causes, risk factors, and sequelae, as well as metadata about them. It's mainly interacted with by importing
+the package and exploring it, and it's used by other Vivarium repositories, especially Vivarium Inputs.
+
+- `Repository <https://github.com/ihmeuw/gbd_mapping>`_
+- `Documentation <https://vivarium.readthedocs.io/projects/gbd-mapping/en/latest/gbd_mapping.html>`_
+
+
+Useful Vivarium Tutorials
++++++++++++++++++++++++++
+
+Each Vivarium repository contains documentation in the form of API references, tutorials, and/or conceptual information.
+or both. These tutorials are especially useful and explain many common tasks but because they are scattered across many
+repositories they can be hard to find. Below is a list of some of the most important pieces of documentation to know
+about.
+
+
+- `The model specification <https://vivarium.readthedocs.io/projects/vivarium/en/latest/concepts/model_specification.html>`_
+- `Running a simulation <https://vivarium.readthedocs.io/projects/vivarium/en/latest/tutorials/running_a_simulation/index.html#running-a-simulation>`_
+- `Exploring a simulation in an interactive setting <https://vivarium.readthedocs.io/projects/vivarium/en/latest/tutorials/exploration.html#exploring-a-simulation-in-an-interactive-setting>`_
+- `Creating and altering a data artifact <https://vivarium-public-health.readthedocs.io/en/latest/tutorials/artifact.html#artifact>`_
+- `The Vivarium Inputs data-pulling functions <https://vivarium.readthedocs.io/projects/vivarium-inputs/en/latest/api_reference/interface.html>`_
```

Comment:
Since this is for researchers, I think it's probably more helpful to link them to the pulling data tutorial (https://vivarium-inputs.readthedocs.io/en/latest/tutorials/pulling_data.html) rather than the API docs